### PR TITLE
Improve dominos order

### PR DESCRIPTION
**Pull request by thorsteinsson**

### CODE REVIEW

File: dominos.pizza [hunk 1]
```diff
@@ -1,8 +1,6 @@
...
 
-* 12" Thin crust - pepperoni, extra pepperoni, extra cheese, chili pepper
-* 12" Italian crust - pepperoni, extra pepperoni, jalapeno, black pepper
-* 12" Italian crust - pepperoni, extra peperoni, ananas, chili pepper
-* 12" 1/2 Italian crust - pepperoni, bacon curl, cream cheese, black pepper
-* 12" 1/2 Italian crust - pepperoni, ground beef, mushrooms, black pepper
-* 2X Coca cola
+* 12" Italian crust - pepperoni, extra pepperoni, extra cheese, chili pepper
+* 12" Italian crust - pepperoni, pinapple, jalapeno, black pepper
+* 12" Italian crust - pepperoni, extra peperoni, pinapple, chili pepper
```
Comment (eirikurn) [inline dominos.pizza]: What is this `pin-apple`? :)

**Comment by eirikurn** [comment]
One comment. Otherwise LGTM

File: dominos.pizza (hunk 1 above)
Comment (thorsteinsson) [inline dominos.pizza]: hehe

**Comment by Flassari** [comment]
Good pizza preferences, like the new changes :+1:

**Pull request merged**